main.go

```diff
@@ -44,6 +44,7 @@ func main() {
 		nodeIP   string
 		nodeEIP  string
 		topdirs  []string
+		user     bool
```

Comment:
I suggest renaming to something like `userSystemd` or something cause `user` isn't obvious about its intents.

Reply:
redone this completely

Reply:
The attribute is still named `user`.

Reply:
Nevermind.